src/main/java/com/parkingapp/parkingservice/domain/vehicle/Vehicle.java

```diff
@@ -0,0 +1,18 @@
+package com.parkingapp.parkingservice.domain.vehicle;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+public class Vehicle {
+    private UUID id;
+    private String brand;
+    private String model;
+    private String color;
+    private String plate;
+    private String country;
```

Comment:
```suggestion
    private Country country;
```

and create enum Country with format: Country(String countryName, String isoCountryName, String iso2Name, int isoCode)

Reply:
Added country enum!